.github/workflows/gradle-build.yml

```diff
@@ -0,0 +1,68 @@
+# This workflow will build a Java project with Gradle and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Java CI with Gradle
+
+on:
+  pull_request:
```

Comment:
keep push too, (on top of PR, and dispatch), include the same filters for push than PR